src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION = 0;
+    private static final int AMOUNT = 1;
+    private static final String REGEX = ",";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private StringBuilder stringBuilder;
+    private File file;
+    private int sumSupply = 0;
+    private int sumBuy = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        readTheFile(fromFileName);
+        countSum();
+        writeTheFile(toFileName);
+    }
+
+    private void readTheFile(String fromFileName) {
```

Comment:
This method should not be void. Return in it what you read from the file

Reply:
Why I can`t use void methods. Is it forbidden or ... I dont understand